src/css/import-resource-tree.css

```diff
@@ -62,6 +62,7 @@
     display: flex;
     align-items: center;
     line-height: 18px;
+    word-break: break-all;
```

Comment:
we should apply ellipsis in the middle, same as for the context

Reply:
Done